mobile/app/components/Avatar.tsx

```diff
@@ -77,20 +86,13 @@ export default function Avatar({ size = 150, onUpload }: Props) {
       ) : (
         <View style={[avatarSize, styles.avatar, styles.noImage]} />
       )}
-      <View>
-        <Button
-          title={uploading ? "Uploading ..." : "Upload"}
-          onPress={uploadAvatar}
-          disabled={uploading}
-        />
-      </View>
-    </View>
+    </Pressable>
   );
 }
 
 const styles = StyleSheet.create({
   avatar: {
-    borderRadius: 5,
+    borderRadius: 999,
```

Comment:
```suggestion
    borderRadius: "100%",
```
But I will survive 999 i guess :D